### PR TITLE
Little fixes

### LINKED ISSUE #9: Update Slack Invite Link
Should change from the expiring invite link to the Heroku app: https://owasp-slack.herokuapp.com/

### PR DESCRIPTION
**Pull request by martinmarsicano**
Updated Slack Heroku invite link ( Fixed #9 )  and some project links, now migrated to the new website.

### CODE REVIEW

File: tab_owasp.md [hunk 1]
```diff
@@ -32,14 +32,14 @@ Debajo se listan los proyectos que se categorizan como proyectos Flagship.
 * [OWASP CSRFGuard Project](https://www.owasp.org/www-project-csrfguard)
 
 ### Documentación
-* [OWASP Application Security Verification Standard Project](https://wiki.owasp.org/index.php/Category:OWASP_Application_Security_Verification_Standard_Project)
+* [OWASP Application Security Verification Standard Project](https://owasp.org/www-project-application-security-verification-standard/)
 * [OWASP Software Assurance Maturity Model (SAMM)](https://owaspsamm.org)
 * [OWASP AppSensor Project](https://wiki.owasp.org/index.php/OWASP_AppSensor_Project)
-* [OWASP Top Ten Project](https://wiki.owasp.org/index.php/Category:OWASP_Top_Ten_Project)
+* [OWASP Top Ten Project](https://owasp.org/www-project-top-ten/)
 * [OWASP Testing Project](https://www.owasp.org/www-project-testing)
```
Comment (kingthorin) [inline tab_owasp.md]:
```suggestion
* [OWASP Web Security Testing Guide](https://owasp.org/www-project-web-security-testing-guide)
```

**Review by kingthorin — COMMENTED** [review]
Slack change looks good, one other minor tweak

**Review by kingthorin — APPROVED** [review]
Outside approval :wink:

**Review by fzipi — APPROVED** [review]
LGTM

**Pull request merged**